client/app/expenses/components/PaymentItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Checkbox } from '@/_components/ui'
+import { UserDisplay } from './UserDisplay'
+import { useUpdatePaymentStatusMutation } from '../hooks/useUpdatePaymentStatus'
+import { Payment } from '../types'
+
+type PaymentItemProps = {
+  item: Payment
+  userId: number
+}
+
+const PaymentItem = ({ item, userId }: PaymentItemProps) => {
+  const { mutate: updatePaymentStatus, isPending } =
+    useUpdatePaymentStatusMutation()
+
+  const handleUpdatePaymentStatus = () => {
+    updatePaymentStatus({
+      paymentId: item.id,
+      isPaid: item.paidDate === null ? true : false,
+    })
+  }
+
+  return (
+    <li className='flex items-center justify-between'>
+      <div className='flex items-center gap-3'>
+        <Checkbox
+          className='h-6 w-6'
+          checked={item.paidDate === null ? false : true}
+          disabled={isPending || item.user.id !== userId}
```

Comment:
I though everyone can click here!

Reply:
Change in a way that if the user is either buyer or payer, they can update
https://github.com/HAK2024/fairy-share/pull/113/commits/369398f9d48ec050c3f2ec49d3281ff2080821d2